### PR TITLE
Use one base template for all HTTP response pages

### PR DESCRIPTION
**Pull request by edds**
Use one base error partial for all response pages rather than two. This
means that we can iterate the look and feel of the one page rather than
having to copy paste code between the two.

### CODE REVIEW

File: app/views/root/_error_page.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<%
+title = "Sorry, we are experiencing technical difficulties (#{status_code} error)" unless local_assigns.include?(:title)
+heading = "Sorry, we are experiencing technical difficulties" unless local_assigns.include?(:heading)
+intro = "Please try again in a few moments." unless local_assigns.include?(:intro)
```
Comment (bradleywright) [inline app/views/root/_error_page.html.erb]: Is it worth moving these conditionals to the place they're rendered at, for clarity? Not a blocking change.

Comment (edds) [inline app/views/root/_error_page.html.erb]: Good idea. I've moved the heading and intro down to their usages.

**Comment by bradleywright** [comment]
Cool. Is this testable?

**Comment by bradleywright** [comment]
Confirmed these are already tested.

**Pull request merged**